Sezai/index.js

```diff
@@ -0,0 +1,121 @@
+//These exercises are aimed at making you more comfortable with using conditionals as well as operators. Use "if" statements to complete the following exercises. Print your results to the console.
+
+//Check if two given integer values are in the range 50 to 99 (this includes both 50 and 99). Print true if either of them are in the range.
+
+
+let num1 = 49;
+let num2 = 100;
+
+if(num1 >=50 && num2 <= 90){
```

Comment:
You have to check if either of them are in the range. so you have to check if every variable is in the range and after that pront if one of them are in the range